test/fixtures/mson/enum-multiple-default.json

```diff
@@ -54,6 +54,19 @@
                       "value": {
                         "element": "enum",
                         "attributes": {
+                          "enumerations": {
+                            "element": "array",
+                            "content": [
+                              {
+                                "element": "string",
+                                "content": "J"
+                              },
+                              {
+                                "element": "string",
+                                "content": "K"
```

Comment:
This matches any `string`. Are you missing `fixed` attribute on the `string` elements above?

Reply:
see general comment https://github.com/apiaryio/drafter/pull/532#issuecomment-374215268